weaviate/collections/classes/internal.py

```diff
@@ -95,6 +95,21 @@ class _Object(Generic[P]):
     vector: Optional[List[float]]
 
 
+@dataclass
+class _MetadataSingleObjectReturn:
+    creation_time_unix: int
+    last_update_time_unix: int
+    is_consistent: Optional[bool]
+
+
+@dataclass
+class _ObjectSingleReturn(Generic[P]):
+    uuid: uuid_package.UUID
+    metadata: _MetadataSingleObjectReturn
+    properties: P
+    vector: Optional[List[float]]
+
+
```

Comment:
Is this needed because many of the fields of `_MetadataQuery` don't make sense when requesting a single object?